docs/en/operation/Admin-CLI.md

```diff
@@ -362,3 +396,158 @@ alluxio.master.audit.logging.enabled
 alluxio.master.ufs.managed.blocking.enabled
 alluxio.master.metastore.inode.inherit.owner.and.group
 ```
+
+### decommissionWorker
+
+The `decommissionWorker` command can be used to take the target workers off-line from the cluster, 
+so Alluxio clients and proxy instances stop using those workers, and therefore they can be killed or restarted gracefully.
+Note that this command will NOT kill worker processes. This command will NOT remove the cache on the workers.
+This command can be typically used for the following use cases:
+1. Perform a graceful rolling restart of all workers in the cluster, where no user requests should fail.
+2. Scale down the cluster without interrupting user I/O workflow.
+
+```shell
+$ ./bin/alluxio fsadmin decommissionWorker --addresses data-worker-0,data-worker-1 [--wait 5m] [--disable]
+```
+The arguments are explained as follows:
+
+`--address/-a` is a required argument, followed by a list of comma-separated worker addresses. Each worker address is `<host>:<web port>`.
+Unlike many other commands which specify the RPC port, we use the web port here because the command will monitor the worker's workload
+exposed at the web port. If the port is not specified, the value in `alluxio.worker.web.port` will be used. Note that `alluxio.worker.web.port`
+will be resolved from the node where this command is run.
+
+`--wait/-w` is an optional argument. This argument defines how long this command waits for the workers to become idle.
+This command returns either when all workers are idle, or when this wait time is up. The default value is `5m`.
+
+`--disable/-d` is an optional argument. If this is specified, the decommissioned workers will not be able to
+register to the master. In other words, a disabled worker cannot join the cluster and will not be chosen for I/O requests.
+This is useful when the admin wants to remove the workers from the cluster. When those disabled workers register,
+the master will reject them but will not kill the worker processes. This is often used in pair with the `enableWorker` command.
+
+The command will perform the following actions:
+
+1. For each worker in the batch, send a decommission command to the primary Alluxio master so 
```

Comment:
Do you think this deserves a dedicated section in [`Administration > Upgrade`](https://docs.alluxio.io/ee/user/stable/en/administration/Upgrade.html)? This `Admin CLI` is mostly focusing on the command and its parameters, it would be better to present the overall process of cluster rolling upgrade in a more mission-centric way.

Reply:
added a few sections to the `Upgrade` page, PTAL